src/pages/AppDetail.svelte

```diff
@@ -5,19 +5,46 @@
   import FilterInput from "../components/FilterInput.svelte";
   import AppAlert from "../components/AppAlert.svelte";
   import NotFound from "../components/NotFound.svelte";
+  import Pagination, {
+    makePages,
+    goToPage,
+  } from "../components/Pagination.svelte";
 
   export let params;
   const URL = `data/${params.app}/index.json`;
   let app;
+  let to;
+  let from;
+  let total;
+  let metrics = [];
+  let pages = [];
+  let paginationData;
+  let lastPage;
+  let currentPage = 1;
   let filteredMetrics;
   let filteredPings;
 
+  function loadPage(args) {
+    let tempPaginationData = goToPage(args.page, pages);
+    from = tempPaginationData.from;
+    to = tempPaginationData.to;
+    filteredMetrics = tempPaginationData.page;
+    currentPage = args.page;
```

Comment:
If you follow my suggestion above, we can probably simplify this function to something like:

```suggestion
    paginationState = goToPage(args.page, pages);
    currentPage = args.page;
```